### PR TITLE
Night Mode + CSS Additions

### LINKED ISSUE #6212: Popular request: create a Night Mode with dark CSS
Here's an example of a "night mode wikipedia app": 

Some campers have created unofficial night mode skins for FCC, and I think we can create a good one that can be toggled on campers' code portfolios.

### LINKED ISSUE #6385: Ability to toggle night mode on and off from options

### PR DESCRIPTION
**Pull request by justincorrigible**
Implementation of night mode and the new hot keys. Massive credit to @raisedadead, and everyone else who helped in the design and creation process. Unfortunately the git history on the initial branch was all over the place so it cannot be used. @benmcmahon100 is the man, man!

Closes #6212
Closes #6385

### CODE REVIEW

**Comment by raisedadead** [comment]
@benmcmahon100 @hallaathrad are the guys who did the heavy lifting though! We gotta keep testing for issues. Can we push this to beta? and get some feedback?

/cc @FreeCodeCamp/issue-moderators

File: client/main.js [hunk 1]
```diff
@@ -534,4 +534,71 @@ $(document).ready(function() {
 
   // keyboard shortcuts: open map
   window.Mousetrap.bind('g m', toggleMap);
+
+  // Night Mode
+  function changeMode() {
+    var newValue = !JSON.parse(localStorage.getItem('nightMode'));
```
Comment (BerkeleyTrue) [inline client/main.js]: This parse needs to be wrapped in a try/catch

File: client/main.js [hunk 2]
```diff
@@ -534,4 +534,71 @@ $(document).ready(function() {
...
+    if (nightModeEnabled) {
+      body.addClass('night');
+    } else {
+      body.removeClass('night');
+    }
+    body.fadeIn('100');
+  }
+
+  if (typeof localStorage.getItem('nightMode') !== 'undefined') {
+    toggleNightMode(JSON.parse(localStorage.getItem('nightMode')));
```
Comment (BerkeleyTrue) [inline client/main.js]: Parse needs a try/catch

File: server/views/layout-wide.jade [hunk 3]
```diff
@@ -8,7 +8,8 @@ html(lang='en')
           include partials/scripts
           block content
   else
-      body.no-top-and-bottom-margins.full-screen-body-background
+      //body.no-top-and-bottom-margins.full-screen-body-background
```
Comment (BerkeleyTrue) [inline server/views/layout-wide.jade]: Remove this line if it's not being used

**Pull request merged**